superduperdb/components/model.py

```diff
@@ -54,7 +54,7 @@ def get(self, k, default=None):
         try:
             return getattr(self, k)
         except AttributeError:
-            return self.kwargs.get(k, default=default)
+            return self.kwargs.get(k, default)
```

Comment:
`dict.get` does not have`default` parameter

Reply:
Gotcha.